Meta/Lagom/Tools/CodeGenerators/JSSpecCompiler/Parser/ParseError.h

```diff
@@ -24,6 +26,7 @@ class ParseError : public RefCounted<ParseError> {
     static NonnullRefPtr<ParseError> create(ErrorOr<String> message, XML::Node const* node);
```

Comment:
perhaps we should be asking the Diagnostic engine to create our errors for us? Then we wouldn't need the "Current file" concept.